README.md

```diff
@@ -10,3 +10,45 @@ single location. The goals of `issue-wanted` are to make it easier for
 programmers of all skill levels to find Haskell projects to contribute to,
 increase the number of contributions to open-source Haskell projects, and
 encourage more programmers to become a part of the Haskell community.
+
+# Testing
```

Comment:
Heh, I forgot to mention, that this `# Testing` header is also redundant because not everything here is about testing.

Reply:
Oh okay got it.